### ISSUE TITLE
Index fulldump in mabxml-elasticsearch

### ISSUE THREAD

**Issue by dr0i** [hbz/lobid-resources#95]
Since #94 is finished we could stop ES on quaoar1. What's missing is the fulldump indexing of `mabxml-elasticsearch` on `gaia-aither` - and the updates must be adapted in the crontab accordingly.

**Comment by fsteeg** [comment]
Created new index in new cluster and triggered indexing with:

```bash
curl --fail -XPOST "http://localhost:7301/hbz01/transform?dir=/files/open_data/open/DE-605/mabxml/&suffix=gz&cluster=gaia-aither&hostname=193.30.112.82&index=hbz01-test" >> logs/processMabxml.sh.$date.log 2>&1 &
```

Set up mabxml-elasticsearch-staging's cron.sh to update hbz01-test index in new cluster.

**Comment by fsteeg** [comment]
Index creation worked, but not enough space on the cluster (if I interpret the output of `df -h` correctly).

Deleting new mabxml index (`mabxml-20160718-1030`), moving this to backlog.

@dr0i I think space will be tight for third normal resources index to be created over the weekend.

**Comment by dr0i** [comment]
The issue of this ticket is solved - fulldump is indexed to gaia-aither cluster. Furthermore, I cleansed both server: Though having two fulldump indices `gaia` has now 140 GiB free hd, `aither` 100 GiB. This is good enough because one fulldump index consumes ~60 GiB, and [it's ensured to have 3 indices at the max](https://github.com/hbz/lobid-resources/blob/master/src/main/java/org/lobid/resources/ElasticsearchIndexer.java#L297).
Closing.
